signing/sign.proj

```diff
@@ -2,6 +2,11 @@
 <Project InitialTargets="SetSigningProperties" DefaultTargets="SignFiles" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildThisFileDirectory)..\dir.props" />
 
+  <PropertyGroup>
+    <IsSignTypeReal Condition="'$(SignType)' == 'Real' OR '$(SignType)' == 'real'">true</IsSignTypeReal>
```

Comment:
For the future, consider:
```
<IsSignTypeReal Condition="'$(SignType.ToLower())' == 'real'>true</IsSignTypeReal>
```
(... or just do the check wherever you've used this property)

Reply:
Actually on further inspection, at least on Windows MSbuild isn't case sensitive for properties!